src/qonnx/converters/qkeras/quantizers.py

```diff
@@ -25,7 +25,7 @@ def _get_quantizer_scale(tensor, quantizer):
     # call the quantizer on the tensor to get its scale
     import numpy as np
 
-    quantized_tensor = quantizer(np.array(tensor).astype(np.float32))
+    quantizer(np.array(tensor).astype(np.float32))
```

Comment:
@vloncar I wasn't 100% sure about this flake8-related change I made. Is it correct that we just are calling the quantizer on the tensor and then grabbing the scale from the quantizer? or is this supposed to return `quantized_tensor.scale` or something?